backbone.layoutmanager.js

```diff
@@ -854,6 +864,11 @@ var LayoutManager = Backbone.View.extend({
     if (options.useRAF === false) {
       Backbone.View.prototype.useRAF = false;
     }
+
+    // Allow underscore to be swapped out
+    if (options._) {
```

Comment:
I added this so I could swap out utility libraries during testing. not sure how useful this will be outside a test suite